components/playlist/common/buildflags/buildflags.gni

```diff
@@ -6,7 +6,7 @@
 import("//build/config/ui.gni")
 
 declare_args() {
-  enable_playlist = toolkit_views || is_android
+  enable_playlist = toolkit_views || is_android || is_ios
```

Comment:
this is effectively `enable_playlist = true` so I would set that now and remove the buildflag in a follow-up

Reply:
but removing the buildflag shouldn't go in this PR, it should be a follow-up